### PR TITLE
pass -tags flag to guru

### PR DESCRIPTION
**Pull request by sillyousu**
`function! go#def#Jump` should pass tags flags to guru.

### CODE REVIEW

File: autoload/go/def.vim [hunk 1]
```diff
@@ -10,8 +10,14 @@ function! go#def#Jump(mode)
 	let old_gopath = $GOPATH
 	let $GOPATH = go#path#Detect()
 
+	let flags = ""
+	if exists('g:go_guru_tags')
+		let tags = get(g:, 'go_guru_tags')
+		let flags .= printf(" -tags %s", tags)
```
Comment (fatih) [inline autoload/go/def.vim]: This should use `=` as we don't append to the string.

**Comment by fatih** [comment]
Thanks @sillyousu for the fix. There is a comment. Once you fix it it's good to go.

**Comment by sillyousu** [comment]
ping @fatih . Fixed.

**Comment by fatih** [comment]
Thanks @sillyousu

**Pull request merged**